src/app/learn/get-started/prereqs/ssl-certificate/index.html

```diff
@@ -213,52 +93,20 @@
     </li>
     <li>
       <p>
-        Under <strong>Servers</strong> on the Certificate Manager, select <strong>Import</strong>.
+        Under <strong>Servers</strong> on the Certificate Manager, select <strong>Add Exception</strong>.
       </p>
     </li>
     <li>
       <p>
-        Open the <sky-code>skyux-server-cert.pem</sky-code> from the <sky-code>.skyux/certs</sky-code> folder in your home directory.
+        Specify https://localhost:8000 as the location, or specify an alternate port if you configured one in the <sky-code>skyuxconfig.json</sky-code> file.
```

Comment:
Not sure if it's worth calling out, but there's a third condition when a developer is running `skyux serve` for more than one SPA.

Reply:
Seems like this should be an AND statement instead of an OR. What do you think of:

```Specify all possible locations. The default location is https://localhost:8000, and you can specify other ports in the <sky-code>skyuxconfig.json</sky-code> file or in additional configuration files.```